docs/docs/configuration/table-configuration.md

```diff
@@ -18,12 +18,12 @@ id: table-configuration
 | `write_compression` | The compression type to use for any storage format that allows compression to be specified. To see which options are available, see [CREATE TABLE AS](https://docs.aws.amazon.com/athena/latest/ug/create-table-as.html). Example: `SNAPPY`.                    | `none`                |
 | `field_delimiter`   | Custom field delimiter. Used when the format is set to `TEXTFILE`. See [CREATE TABLE AS](https://docs.aws.amazon.com/athena/latest/ug/create-table-as.html). Example: `','`                                                                                     | `none`                |
 | `table_properties`  | Additional table properties to add to the table. Valid for Iceberg only. Example: `{'optimize_rewrite_delete_file_threshold': '2'}`                                                                                                                             | `none`                |
-| `lf_tags`           | Lake Formation tags for metadata access control, to associate to the table. Example: `{"tag1":{"tag1": "value1", "tag2": "value2"}`                                                                                                                             | `none`                |
-| `lf_tags_columns`   | Lake Formation tags for metadata access control, to associate to columns. Example: `{"tag1": {"value1": ["column1": "column2"]}}`                                                                                                                               | `none`                |
+| `lf_tags_config`    | Lake Formation tags for metadata access control, to associate to the table. See detailed instructions [here](./lakeformation)                                                                                                                                   | `none`                |
```

Comment:
Maybe worth adding a conditional block to show this only on 1.5.8 ?

Reply:
I'm not sure we can do it inside markdown table. For now I've add note about the first adapter version for this config. It seems we can properly show | hide versioned configs using different docs according to the doc https://docusaurus.io/docs/next/versioning#overview , however this seems to be overkill for now. Did I miss smth about docusaurus versioning config ?

Reply:
You can do it, here is an example here: https://github.com/dbt-athena/dbt-athena.github.io/blob/main/docs/docs/configuration/materializations/hive.md?plain=1#L13

Reply:
yes, but in case of markdown table we should put this tag to every cell and we'll have empty rows in this case for older versions. It does not look like good I guess

Reply:
however I've managed to add specific section with LF config and it's now versioned